packages/nimble-components/src/table-column/number-text/tests/table-column-number-text-matrix.stories.ts

```diff
@@ -0,0 +1,70 @@
+import type { StoryFn, Meta } from '@storybook/html';
```

Comment:
I think we need a usage note for round-to-integer stating that our numbers are represented as treated as 64-bit floats so notably cannot represent large 64-bit integers losslessly. And maybe linking to the mdn MAX_SAFE_INTEGER docs.

We'll need to add an mdx page so don't feel strongly if in this PR or follow-up.